src/query.js

```diff
@@ -5,14 +5,15 @@
  * @see https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Query.html
  * @module Query
  */
-const { bind, pipeP, curry, compose } = require('ramda');
-const { unwrapAll } = require('./wrapper');
+const { bind, curry, compose } = require('ramda');
+const { unwrapAll, unwrapOverAll } = require('./wrapper');
 const addTableName = require('./table-name');
 
 /**
  * @private
  */
-const createQuery = query => pipeP(query, unwrapAll('Items'));
+const createQuery = query => (params, options = {}) =>
```

Comment:
I see that the original `query` function has an arity of `2`.

```sh
> const clientWrapper = new DynamoDBWrapper({});
undefined
> clientWrapper.query.length
2
```

We should probably keep that and avoid the default `= {}` value for `options`.

```js
options && options.raw ? unwrapOverAll('Items') : unwrapAll('Items')
```